core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -33,6 +33,10 @@ THE SOFTWARE.
     <link rel="alternate" title="Jenkins:${it.viewName} (all builds) (RSS 2.0)" href="${rootURL}/${it.url}rssAll?flavor=rss20" type="application/rss+xml" />
     <link rel="alternate" title="Jenkins:${it.viewName} (failed builds)" href="${rootURL}/${it.url}rssFailed" type="application/rss+xml" />
     <link rel="alternate" title="Jenkins:${it.viewName} (failed builds) (RSS 2.0)" href="${rootURL}/${it.url}rssFailed?flavor=rss20" type="application/rss+xml" />
+
+    <l:css src="jsbundles/remodal.css" />
+    <l:js src="jsbundles/add-item.js" />
+    <l:css src="jsbundles/add-item.css" />
```

Comment:
I tried to add .new-job-popup style to add-item.css
However, every time I rebuild jenkins, all my changes from my file are removed. Could you explain why? It is strange to me since that was not the case with add-item.js